examples/AN00162_i2s_loopback_demo/doc/rst/AN00162.rst

```diff
@@ -280,6 +280,18 @@ References
 
    http://www.xmos.com/published/xmos-programming-guide
 
+ * XMOS |I2S| Library
```

Comment:
I think this should read I2S/TDM library to be consistent with the readme, i.e.
 * XMOS |I2S|/TDM library